easy-connect.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/easy-connect/#bf821b0695ccada3620a6091503eb53a2a1ac3c8
+https://github.com/ARMmbed/easy-connect/#adb235d929815d92083aa2abeed9f29b0386b2c3
```

Comment:
Update with latest `easy-connect` hash code (once merged in).

Reply:
Yep!